networking/synthetic/stac-prepare-nodes.py

```diff
@@ -82,14 +82,28 @@ def parse_args():
                         dest='iface_name',
                         help='interace name on nodes through which stac test traffic will run')
 
+    parser.add_argument('-t',
+                        '--test-file',
+                        required=True,
+                        dest='test_file',
+                        help='a yaml test file')
+
+    parser.add_argument('-p',
+                        '--tracing',
+                        required=False,
+                        dest='pbd_tracing',
+                        help='profile the app with pdb tracing')
+
     return parser.parse_args()
 
 
 def main():
     args = parse_args()
-    nettest = NetworkTest('stac-test.yaml')
```

Comment:
@mrsiano IMO, only file name correction, `stac-prepare-nodes.yaml` instead of `stac-test.yaml`, should be enough.
I think we should not make it configurable to keep knobs to a minimal for ease of use. I see this as an internal file which user should not be aware about. Any test case related configuration is maintained at github repo, which is passed as a command line variable. And because that repo path will change from user to user, so it makes sense to pass that as a variable. This file name will not change like that.